guvnor-structure/guvnor-structure-api/src/main/java/org/guvnor/structure/repositories/GitMetadataStore.java

```diff
@@ -37,6 +37,14 @@ void write( String name,
                 String origin );
 
     /**
+     * Writes the meta information about a repository.
+     * @param name The name of the repository in organizationalUnit/repositoryName format
```

Comment:
can you clarify the javadoc on repository name? i mean.. it can be anything, right?